app/assets/stylesheets/themes/default/_ad.scss

```diff
@@ -6,6 +6,7 @@
 
 .promotion .ad {
   text-align: center;
+  margin-top: 20px;
   @include media($smaller-tablet) {
     img {
```

Comment:
`img` style seems not to work because the image will be inserted by adsense/dfp and placed in iframe :dizzy_face: 
So we must center the container instead of inline elements.